samples/java_springboot/24.bot-authentication-msgraph/README.md

```diff
@@ -101,18 +112,16 @@ This error may confusingly present itself if either of the following are true:
 
 ## Further reading
 
-- [Bot Framework Documentation](https://docs.botframework.com)
+- [Spring Boot](https://spring.io/projects/spring-boot)
+- [Maven Plugin for Azure App Service](https://docs.microsoft.com/en-us/java/api/overview/azure/maven/azure-webapp-maven-plugin/readme?view=azure-java-stable)
```

Comment:
This link points to a 404 - page not found, [check it](https://docs.microsoft.com/en-us/java/api/overview/azure/maven/azure-webapp-maven-plugin/readme?view=azure-java-stable)

Reply:
Good catch, I'm pushing the fix